pluginLauncher/lib/source/DobbyRdkPluginUtils.cpp

```diff
@@ -146,44 +153,31 @@ std::string DobbyRdkPluginUtils::getContainerId() const
  */
 bool DobbyRdkPluginUtils::getContainerNetworkInfo(ContainerNetworkInfo &networkInfo)
 {
-    // Attempt to find the file
-    const std::string containerId = getContainerId();
-    if (containerId.empty())
-    {
-        AI_LOG_ERROR_EXIT("Could not get container network info - could not get ID");
-        return false;
-    }
-
-    const std::string fileName = ADDRESS_FILE_PREFIX + getContainerId();
-
-    struct stat buffer;
-    if (stat(fileName.c_str(), &buffer) != 0)
-    {
-        AI_LOG_WARN("Could not get container network info - file %s does not exist. Has the network plugin run yet?", fileName.c_str());
-        return false;
-    }
-
     // Parse the file
-    const std::string addressFileStr = readTextFile(fileName);
+    std::string filePath = ADDRESS_FILE_DIR + getContainerId();
+    const std::string addressFileStr = readTextFile(filePath);
     if (addressFileStr.empty())
     {
-        AI_LOG_ERROR_EXIT("failed to get IP address and veth name assigned to"
-                          "container from %s",
-                          fileName.c_str());
+        AI_LOG_ERROR_EXIT("failed to get IP address and veth name assigned to container from %s",
+                          filePath.c_str());
         return false;
     }
 
-    const std::string ip = addressFileStr.substr(0, addressFileStr.find("/"));
+    const std::string ipStr = addressFileStr.substr(0, addressFileStr.find("/"));
+    const in_addr_t ip = std::stoi(ipStr);
 
     // check if string contains a veth name after the ip address
-    if (addressFileStr.length() <= ip.length() + 1)
+    if (addressFileStr.length() <= ipStr.length() + 1)
     {
-        AI_LOG_ERROR("failed to get veth name from %s", fileName.c_str());
+        AI_LOG_ERROR("failed to get veth name from %s", filePath.c_str());
         return false;
     }
 
-    networkInfo.ipAddress = ip;
-    networkInfo.vethName = addressFileStr.substr(ip.length() + 1, addressFileStr.length());
+    networkInfo.containerId = basename(filePath.c_str());
+    networkInfo.ipAddressRaw = ip;
+    networkInfo.ipAddress = ipAddressToString(htonl(ip));
```

Comment:
I thought that you just had ipStr with sting version IP address, if this is because you need special formatting, then fine if not then just use ipStr here. Same for getNetworkInfo.